provisioning/device/samples/ProvisioningDeviceClientX509/readme.md

```diff
@@ -42,8 +42,7 @@ to
 ```C# 
     var myCertificate = new X509Certificate2("myCertificate.pfx", "mypassword");
     var myChain = new X509Certificate2Collection();
-    myChain.Import("myChain.p7b");
```

Comment:
@nberdy any reason this didn't work for you? What was the error?

Reply:
There's no .p7b file with that name, and I never create one in any of the steps (nor am I instructed to do so). As far as I can tell, there's no need for this line.

Reply:
(Nor do I know how to debug why I have no .p7b file or where to get one)

Reply:
Good point regarding the temporary scripts. 
We should create a p7b or comment this line saying that if a collection of intermediate certificates is available we provide the functionality to import them in the correct store.